src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/OveruseEstimator.java

```diff
@@ -86,7 +86,8 @@
 
     private double slope;
 
-    private final List<Double> tsDeltaHist = new LinkedList<>();
+    private final double[] tsDeltaHist = new double[kMinFramePeriodHistoryLength];
+    private int tsDeltaHistIdx = 0;
```

Comment:
I think the part Idx of the name tsDeltaHistIdx could be more meaningful. Index is generic enough to not bring much but the fact that it's an index (because the fact that it's related to the array tsDeltaHist is represented by the prefix tsDeltaHist). If I'm reading your source code right, you're implementing a circular buffer and tsDeltaHistIdx is the tail (or the insertion head but that's another discussion), Wouldn't it be better to reflect more meaning in the name?